### ISSUE TITLE
Relations cause exceptions.

### ISSUE THREAD

**Issue by tscolin** [edbmods/EdBPrepareCarefully#248]
4 colonists, blank template.  Adding a relation results in unusable prepare carefully.  Log follows, issue repeats infinitely until spam blocked. EdBPrepareCarefully is the only loaded mod.

```C#
RimWorld 1.0.2059 rev1005
Verse.Log:Message(String, Boolean)
RimWorld.VersionControl:LogVersionNumber()
Verse.Root:CheckGlobalInit()
Verse.Root:Start()
Verse.Root_Entry:Start()

Exception filling window for EdB.PrepareCarefully.DialogSelectParentChildPawn: System.InvalidOperationException: Nullable object must have a value.
  at System.Nullable`1[System.Int32].get_Value () [0x00000] in <filename unknown>:0 
  at EdB.PrepareCarefully.CustomPawn.get_FullName () [0x00000] in <filename unknown>:0 
  at EdB.PrepareCarefully.DialogSelectParentChildPawn+<>c__DisplayClass92_0.<Resize>b__2 (EdB.PrepareCarefully.CustomPawn pawn, Rect rect, Metadata metadata) [0x00000] in <filename unknown>:0 
  at EdB.PrepareCarefully.WidgetTable`1[EdB.PrepareCarefully.CustomPawn].DrawRow (Single cursor, EdB.PrepareCarefully.CustomPawn row, Int32 index) [0x00000] in <filename unknown>:0 
  at EdB.PrepareCarefully.WidgetTable`1[EdB.PrepareCarefully.CustomPawn].Draw (IEnumerable`1 rowGroups) [0x00000] in <filename unknown>:0 
  at EdB.PrepareCarefully.DialogSelectParentChildPawn.DoWindowContents (Rect inRect) [0x00000] in <filename unknown>:0 
  at Verse.Window+<WindowOnGUI>c__AnonStorey0.<>m__0 (Int32 x) [0x00000] in <filename unknown>:0 
Verse.Log:Error(String, Boolean)
Verse.<WindowOnGUI>c__AnonStorey0:<>m__0(Int32)
UnityEngine.GUI:CallWindowDelegate(WindowFunction, Int32, Int32, GUISkin, Int32, Single, Single, GUIStyle)

Alignment was LowerLeft at end of frame.
Verse.Log:Error(String, Boolean)
Verse.Log:ErrorOnce(String, Int32, Boolean)
Verse.Text:StartOfOnGUI()
Verse.UIRoot:UIRootOnGUI()
Verse.UIRoot_Entry:UIRootOnGUI()
Verse.Root:OnGUI()

Exception filling window for EdB.PrepareCarefully.DialogSelectParentChildPawn: System.InvalidOperationException: Nullable object must have a value.
  at System.Nullable`1[System.Int32].get_Value () [0x00000] in <filename unknown>:0 
  at EdB.PrepareCarefully.CustomPawn.get_FullName () [0x00000] in <filename unknown>:0 
  at EdB.PrepareCarefully.DialogSelectParentChildPawn+<>c__DisplayClass92_0.<Resize>b__2 (EdB.PrepareCarefully.CustomPawn pawn, Rect rect, Metadata metadata) [0x00000] in <filename unknown>:0 
  at EdB.PrepareCarefully.WidgetTable`1[EdB.PrepareCarefully.CustomPawn].DrawRow (Single cursor, EdB.PrepareCarefully.CustomPawn row, Int32 index) [0x00000] in <filename unknown>:0 
  at EdB.PrepareCarefully.WidgetTable`1[EdB.PrepareCarefully.CustomPawn].Draw (IEnumerable`1 rowGroups) [0x00000] in <filename unknown>:0 
  at EdB.PrepareCarefully.DialogSelectParentChildPawn.DoWindowContents (Rect inRect) [0x00000] in <filename unknown>:0 
  at Verse.Window+<WindowOnGUI>c__AnonStorey0.<>m__0 (Int32 x) [0x00000] in <filename unknown>:0 
Verse.Log:Error(String, Boolean)
Verse.<WindowOnGUI>c__AnonStorey0:<>m__0(Int32)
UnityEngine.GUI:CallWindowDelegate(WindowFunction, Int32, Int32, GUISkin, Int32, Single, Single, GUIStyle)

Reached max messages limit. Stopping logging to avoid spam.
Verse.Log:Warning(String, Boolean)
Verse.Log:PostMessage()
Verse.Log:Error(String, Boolean)
Verse.<WindowOnGUI>c__AnonStorey0:<>m__0(Int32)
UnityEngine.GUI:CallWindowDelegate(WindowFunction, Int32, Int32, GUISkin, Int32, Single, Single, GUIStyle)
```

**Comment by Valerate** [comment]
I just came to report the same issue

**Comment by queeb3** [comment]
having the same problem
https://pastebin.com/1BK7RauF

**Comment by edbmods** [comment]
This issue was resolved with the 1.0.11 release.
